opsdroid/helper.py

```diff
@@ -325,3 +325,30 @@ def register_json_type(type_cls, fields, decode_fn):
         dct["hour"], dct["minute"], dct["second"], dct["microsecond"]
     ),
 )
+
+
+def lookup_target(event, aliases):
```

Comment:
because this is matrix specific I think it should go on the matrix connector as a method. We could then implement a similar one for slack to convert from IDs like `C128371890` to `#general`. The matcher would then just call the method on whatever connector is attached to the event.